test/common/http/conn_manager_utility_test.cc

```diff
@@ -68,7 +68,8 @@ TEST_F(ConnectionManagerUtilityTest, ShouldTraceRequest) {
         Http::AccessLog::ResponseFlag::NoHealthyUpstream,
         Http::AccessLog::ResponseFlag::UpstreamRequestTimeout,
         Http::AccessLog::ResponseFlag::UpstreamOverflow,
-        Http::AccessLog::ResponseFlag::FaultInjected, Http::AccessLog::ResponseFlag::NoRouteFound};
+        Http::AccessLog::ResponseFlag::DelayInjected, Http::AccessLog::ResponseFlag::FaultInjected,
```

Comment:
"So, is the intent that you just want to see why calls didn't arrive at Envoy2 ?" - yup.
The intent is to make sure that we trace either on Envoy1 or on Envoy2 (otherwise we'll lose the last step). When you look at trace you do not want to have missing hop. (there is no alarming but rather when you inspect the trace).
I'm actually inclined to have DI traced as well (as this will give better idea why request took that long on a trace), in this case we might have egress span as well as ingress span but that should be fine.